### DISCUSSION TITLE
Please Help me to trouble shoot

### DISCUSSION THREAD

**Discussion by liujing007008** [Chia-Network/chia-blockchain#6997] [Support]
How can I do with this?
![Uploading EF0B5F3D-BAC8-4F71-96DC-7F56A904E2EB.jpeg…]()

**Comment by gromran** [comment]
Help with posting images correctly?

**Reply by liujing007008** [reply]
![image](https://user-images.githubusercontent.com/51757841/123610766-7acb2f00-d833-11eb-9a24-12feef08b95b.jpeg)

**Reply by gromran** [reply]
long lookup time.
slow hdd, slow internal connection, sleep mode etc.

**Comment by BrandtH22** [comment]
Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
